src/components/Home/HomePage.js

```diff
@@ -0,0 +1,65 @@
+                        title="Rave"
```

Comment:
This is obviously temporary.  Figured we could finalize image assets/styling stuff once everything is linked up, so just used this random image from google image search as a placeholder.